src/components/Subheading/README.md

```diff
@@ -46,12 +46,27 @@ Subheadings should follow the content guidelines for [headings and subheadings](
 
 ## Examples
 
+### Typographic subheading
+
+Used for the title of any sub-sections in top-level page sections.
+
+```jsx
+<Subheading>Accounts</Subheading>
+```
+
 ### Subheading in a card
 
 Use to structure content in a card.
 
 ```jsx
-<Subheading>Accounts</Subheading>
+<Card title="Online store dashboard">
+  <Card.Section>
+    <Subheading>Products</Subheading>
+  </Card.Section>
+  <Card.Section>
+    <Subheading>Orders</Subheading>
+  </Card.Section>
+</Card>
```

Comment:
TLDR: Should we remove all mentions of `Card` from `Subheading` documentation?

Reason: Title on Card.Section implements this for us.

<hr>

I'm going back and forth here regarding the description of this component.

> Generally, sections of a card use subheadings for their titles.

But the `Card.Section` accepts a `title` prop which also implements `Subheading` if the title is a string like so,

https://github.com/Shopify/polaris-react/blob/21be9c8891e5652a479b8c700c40484ee862561e/src/components/Card/components/Section/Section.tsx#L37-L38

One thing I noticed is that the Card `Section` styles whatever its title is so that it has space away from the content that exists within the section,

https://github.com/Shopify/polaris-react/blob/21be9c8891e5652a479b8c700c40484ee862561e/src/components/Card/Card.scss#L66-L67

If you use the `Subheading` component yourself, like the documentation suggests, you need to implement this styling yourself using a Stack, css, etc.

Reply:
> Should we remove all mentions of Card from Subheading documentation?

I think we probably should. I have a feeling it's because this prop was added after and the example wasn't updated.

Reply:
As a rule of thumb examples should always be as simple as possible. Some cases require more complexity, e.g. `Form` needs form elements to have a working example or frame needing top/navigation/etc.